prime-angular-frontend/src/app/modules/site-registration/pages/overview/overview.component.ts

```diff
@@ -97,4 +110,27 @@ export class OverviewComponent implements OnInit {
       this.showSubmission = !this.site.submittedDate;
     }
   }
+
+  /**
+   * @description
+   * Infer whether user was required to accept an Organization Agreement.
+   * User saw an Organization Agreement if the current Site has a care setting that is unique
+   * for that Organization.
+   */
+  private determineOrgAgreementRequired$(): Observable<SiteListViewModel[]> {
```

Comment:
This method naming doesn't reflect what it is returning. I would assume this would return a bool, when instead it returns a SiteListViewModel[]. Is this return relevant? Since i see we just ignore the return above.

As below it looks like we just use this method to increment a private class variable that doesn't seem intuitive. 

Maybe just making this method return true/false and containing all the logic instead of splitting it across mutliple methods and hiding some of the meaning within. That way above we just call 

```
if (this.determineOrgAgreementRequired()) {
   do x;
} else {
   do y
}

private determineOrgAgreementRequired(): boolean {}
```

Reply:
Completely agree @hogmann .  However I was having problems getting this to work with RxJs so I ended up with the member variable hack to get functionality to work.  I'll join the Zoom workspace tomorrow to get your assistance  :-)

Reply:
Addressed in latest commit 68fb52e